federatedscope/core/monitors/monitor.py

```diff
@@ -376,6 +376,24 @@ def format_eval_res(self,
                                 all_res.size // 10]
                             new_results[f"{key}_top_decile"] = all_res[
                                 all_res.size * 9 // 10]
+                            # log more fairness metrics
+                            
+                            # min and max
+                            new_results[f"{key}_min"] = np.mean(all_res[0])
+                            new_results[f"{key}_max"] = np.mean(all_res[-1])
+
+                            # best and worst 10%
+                            new_results[f"{key}_worst10%"] = np.mean(all_res[:
```

Comment:
similar to the above question.  if there is no other special consideration, the best/worst10% will be the same as the metric top/bottom_decile that have been calculated in line 375/377

Reply:
Besides, the term "worst"/"best" is conditioned on the metric. We may need to maintain a flag such as `the_larger_the_better`